tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -137,15 +137,17 @@ def handle_user(self, request, id_token, decoded_token_data):
             # user's `sub` UUID from the decoded payload, with which we will
             # authenticate later.
             initial_user = CustomAuthentication.authenticate(username=email)
+            logging.debug("initial_user obj:{}".format(initial_user))
```

Comment:
resolves an issue with login for those that are set to django superusers cc @ttran-hub